resources/test.js

```diff
@@ -0,0 +1 @@
+window.onload = (event) => {
```

Comment:
Please move all the script files inside a folder for a better folder structure

Reply:
Script files moved to a common folder.